distributed/scheduler.py

```diff
@@ -818,6 +818,19 @@ class TaskGroup:
         "last_worker_tasks_left",
     ]
 
+    name: str
+    prefix: TaskPrefix | None
+    states: dict[str, int]
+    dependencies: set[TaskGroup]
+    nbytes_total: int
+    duration: float
+    types: set[str]
+    start: float
+    stop: float
+    all_durations: defaultdict[str, float]
+    last_worker: WorkerState | None
+    last_worker_tasks_left: int
+
```

Comment:
This is more thoroughly dealt with in https://github.com/dask/distributed/pull/6253